ui/page/governance/agenda_vote_modal.go

```diff
@@ -0,0 +1,307 @@
+package governance
+
+import (
+	"sort"
+
+	"gioui.org/font/gofont"
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/widget"
+	"gioui.org/widget/material"
+
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type agendaVoteModal struct {
+	*load.Load
+	modal decredmaterial.Modal
+
+	LiveTickets []*dcrlibwallet.Transaction
+
+	agenda    *dcrlibwallet.Agenda
+	isVoting  bool
+	loadCount int
+
+	consensusPage *ConsensusPage
+
+	walletSelector    *WalletSelector
+	ticketSelector    *ticketSelector
+	spendingPassword  decredmaterial.Editor
+	materialLoader    material.LoaderStyle
+	items             map[string]string //[key]str-key
+	itemKeys          []string
+	defaultValue      string // str-key
+	initialValue      string
+	currentValue      string
```

Comment:
What items, what values, what keys? These are extremely generic, can never deduce what purpose these variables serve from just looking at the variable names and types.